en/sensor/pmw3901.md

```diff
@@ -182,3 +185,6 @@ The board doesn't include a distance sensor onboard so you will need to use an [
 <!-- note, this will be set using SENS_TFLOW_CFG
 Wiring also required.
 -->
+In addition to the general [PX4 configuration](#px4-configuration) you must also set the parameter [SENS_TFLOW_CFG](../advanced_config/parameter_reference.md#SENS_TFLOW_CFG) to the value of the UART port you connected (e.g. if the sensor is connected to `TELEM 2` then set `SENS_TFLOW_CFG=102`.
+For more information see Serial Port Configuration](../peripherals/serial_configuration.md).
```

Comment:
[Serial Port Configuration]
missing `[`

Reply:
@bys1123 I fixed this for you.